providers/aws/resources/aws.lr

```diff
@@ -777,6 +777,14 @@ private aws.iam.user @defaults("arn name") {
   groups() []string
   // List of access keys metadata associated with the user
   accessKeys() []dict
+  // Login profile for the user
+  loginProfile() aws.iam.loginProfile
+}
+
+// AWS IAM login profile for a user
+private aws.iam.loginProfile @defaults("createDate") {
+  // Time when the login profile was created
+  createDate time
```

Comment:
Can we make this createdAt?  AWS is super inconsistent in this API and we've mostly just copied their API. Lately I've been trying to make them all createdAt since that was the most common use in the SDK. It'll really help things to be predictable for users and avoid us having breaking changes later:

Here's the current numbers:

- createdAt: 17
- createdTime: 6
- createdDate: 1

Reply:
awesome suggestion, we'll get it adjusted